alien4cloud-tosca/src/main/java/alien4cloud/tosca/parser/impl/ErrorCode.java

```diff
@@ -72,6 +72,7 @@
     INVALID_NODE_TEMPLATE_NAME,
     /** A node type a defined in other Csar **/
     TOSCA_TYPE_ALREADY_EXISTS_IN_OTHER_CSAR,
-    /** version conflict between a transitive and a direct dependency **/
```

Comment:
Not intentional to remove this ? DEPENDENCY_VERSION_CONFLICT is not commented/explained anymore.